src/css.js

```diff
@@ -117,43 +117,29 @@ function augmentWidthOrHeight( elem, name, extra, isBorderBox, styles ) {
 
 function getWidthOrHeight( elem, name, extra ) {
 
-	// Start with offset property, which is equivalent to the border-box value
-	var val,
-		valueIsBorderBox = true,
+	// Start with computed style
+	var valueIsBorderBox,
 		styles = getStyles( elem ),
+		val = curCSS( elem, name, styles ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
-	// Support: IE <=11 only
-	// Running getBoundingClientRect on a disconnected node
-	// in IE throws an error.
-	if ( elem.getClientRects().length ) {
-		val = elem.getBoundingClientRect()[ name ];
+	// Fall back to uncomputed css if necessary
+	if ( val < 0 || val == null ) {
```

Comment:
It seems I forgot to simplify it when I migrated the code to use `getBoundingClientRect`.

That said, this is related to this PR - in my PR I should have just changed `val <= 0 || val == null` to `val === null`as the code inside would be used in those cases, only the `val <= 0` condition was never true. In your PR this becomes true dead code and since you even had to change this condition for the new logic to work, I think this should be removed here.